node/src/components/consensus/era_supervisor/era.rs

```diff
@@ -37,13 +36,13 @@ impl EraId {
     }
 
     /// Returns an iterator over all eras that are still bonded in this one, including this one.
-    pub(crate) fn iter_bonded(&self) -> impl Iterator<Item = EraId> {
-        (self.0.saturating_sub(BONDED_ERAS)..=self.0).map(EraId)
+    pub(crate) fn iter_bonded(&self, bonded_eras: u64) -> impl Iterator<Item = EraId> {
+        (self.0.saturating_sub(bonded_eras)..=self.0).map(EraId)
     }
 
     /// Returns an iterator over all eras that are still bonded in this one, excluding this one.
-    pub(crate) fn iter_other_bonded(&self) -> impl Iterator<Item = EraId> {
-        (self.0.saturating_sub(BONDED_ERAS)..self.0).map(EraId)
+    pub(crate) fn iter_other_bonded(&self, bonded_eras: u64) -> impl Iterator<Item = EraId> {
```

Comment:
I guess we might as well rename those to `iter` and `iter_other`, since the number of eras is now a parameter. And the parameter should just be called `count`, too.

Reply:
Done